File: master-thesis-config-2/mtc-2-nodejs-algorithm/src/algorithms/road-plan/logic/coordinate.ts
class Coordinate {
  latitude: number;
  longitude: number;
  address: string;

  /**
   *  Coordinate Constructor
   */
  constructor(latitude: number, longitude: number, address: string) {
    this.latitude = latitude;
    this.longitude = longitude;
    this.address = address;
  }

  distance = (coordinate: Coordinate) => {
    return Math.sqrt(
      Math.pow(coordinate.latitude - this.latitude, 2) +
        Math.pow(coordinate.longitude - this.longitude, 2)
    );
  };
}

export default Coordinate;

File: master-thesis-config-6/mtc-6-dotnet/mysql2/mysql/Domain/Algorithms/Models/Coordinate.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Reflection.Metadata;
using Domain.Models;

namespace Domain.Algorithms.Models
{
    [NotMapped]
    public class Coordinate
    {
        public int Order { get; set; } = 0;
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string Address { get; set; }
    }
}